jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/security-context.xml

```diff
@@ -86,7 +86,6 @@
     <security:intercept-url pattern="/pages/create/**" access="isAuthenticated()"/>
 
     <security:intercept-url pattern="/poll/**" access="isAuthenticated()"/>
-    <security:intercept-url pattern="/search/index/rebuild" access="hasIpAddress('127.0.0.1')"/>
```

Comment:
Why don't you restrict the URL instead of restricting within the service itself?